File: eform-client/src/app/common/models/security/group-permissions/eform/eforms-permissions.model.ts
export class EformsPermissionsModel {
  total: number;
  groupName: string = '';
  groupId: number;
  eformsList: Array<EformPermissionsModel> = [];
}

export class EformPermissionsModel {
  eformInGroupId: number;
  templateId: number;
  label: string;
  createdAt: Date;
  permissionTypes: Array<EformPermissionsTypeModel> = [];
}

export class EformPermissionsTypeModel {
  name: string;
  permissions: Array<EformPermissionModel> = [];
}

export class EformPermissionModel {
  id: number;
  eformPermissionId: number;
  permissionName: string;
  claimName: string;
  permissionTypeId: number;
  permissionType: string;
  isEnabled: boolean;
}

File: eFormAPI/eFormAPI.Web/Infrastructure/Models/EformPermissions/EformsPermissionsModel.cs
using System.Collections.Generic;

namespace eFormAPI.Web.Infrastructure.Models.EformPermissions
{
    public class EformsPermissionsModel
    {
        public int Total { get; set; }
        public int GroupId { get; set; }
        public string GroupName { get; set; }

        public List<EformPermissionsModel> EformsList { get; set; }
            = new List<EformPermissionsModel>();
    }
}